packages/npm2jsr/src/lib/npm2jsr.ts

```diff
@@ -306,7 +306,7 @@ export const npmToJsrMapping: Map<string, JsrPackageInfo> = new Map<
     '@supabase/supabase-js',
     {
       jsrPackage: '@supabase/supabase-js',
-      minimumVersion: '2.43.2',
+      minimumVersion: '2.58.0',
```

Comment:
I think this should actually be 85 as well not 58 
```suggestion
      minimumVersion: '2.85.0',
```